pkg/controller/networkpolicy/store/addressgroup.go

```diff
@@ -162,6 +162,22 @@ func NewAddressGroupStore() storage.Interface {
 			// ag.Selector.Namespace == "" means it's a cluster scoped group, we index it as it is.
 			return []string{ag.Selector.Namespace}, nil
 		},
+		PodIndex: func(obj interface{}) ([]string, error) {
+			ag, ok := obj.(*types.AddressGroup)
+			if !ok {
+				return []string{}, nil
+			}
+			keys := make([]string, 0)
+			if ag.Pods != nil {
```

Comment:
I don't think you need this check. `Pods` is a map and you can iterate over a nil map in go: https://play.golang.org/p/b8VcbthDcxi